### DISCUSSION TITLE
Welcome to Telegram-Leecher Discussions!

### DISCUSSION THREAD

**Discussion by XronTrix10** [XronTrix10/Telegram-Leecher#15] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rvaspentertainment** [comment]
Help me to create only leech bot

**Reply by XronTrix10** [reply]
Follow the tutorial and if get stuck, ask us on our channel
